apps/studio/src/schemas/page.ts

```diff
@@ -12,8 +12,8 @@ export const updatePageSchema = getEditPageSchema.extend({
   // in which case this is a no-op.
   // We are ok w/ this because it doesn't
   // incur any db writes
-  parentId: z.string().nullable(),
-  pageName: z.string().nullable(),
+  parentId: z.string().optional(),
+  name: z.string().optional(),
```

Comment:
hmm i thought following from the slack thread, there is going to be a different api call for this? no ah?

Reply:
that was for folders - the change made as that previously `id` (pkey in db) was optional but now it's not. 

in here, `id` is required